src/Requests/MailchimpRequest.php

```diff
@@ -451,6 +451,10 @@ public function reset()
         $apikey = $this->apikey;
         $request_vars = get_object_vars($this);
         foreach ($request_vars as $key => $value) {
+            if (in_array($key, ['success_callback', 'failure_callback'], true)) {
+                continue;
```

Comment:
Its needed to preserve assigned callbacks for 2+ method calls on the same Mailchimp instance

Reply:
This makes sense to me.